examples/todo/src/datasources/geocoder.datasource.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/example-todo
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {inject} from '@loopback/core';
+import {juggler, DataSource} from '@loopback/repository';
+const config = require('./geocoder.datasource.json');
+
+export class GeocoderDataSource extends juggler.DataSource {
+  static dataSourceName = 'geocoder';
+
+  constructor(
+    @inject('datasources.config.geocoder', {optional: true})
+    dsConfig: DataSource = config,
+  ) {
+    dsConfig = Object.assign({}, dsConfig, {
+      // A workaround for the current design flaw where inside our monorepo,
+      //   packages/repository-proxy/node_modules/loopback-datasource-juggler
```

Comment:
did you mean `packages/repository/node_modules` here instead of `packages/repository-proxy/node_modules`?